BookFinder/build/build.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+require('./check-versions')()
+
+process.env.NODE_ENV = 'production'
```

Comment:
The NODE_ENV should be currently set to 'development' to provide better logs and facilitate development.